packages/core/core/src/Parcel.js

```diff
@@ -166,33 +181,35 @@ export default class Parcel {
   async run(): Promise<IBundleGraph<INamedBundle>> {
```

Comment:
Not really related to this change but it might be nice to be able to pass an `AbortSignal` here...